package.json

```diff
@@ -47,6 +47,7 @@
     "tsconfig.json"
   ],
   "dependencies": {
+    "gray-matter": "^4.0.3",
```

Comment:
Can we potentially use an optional dependency somehow here? Nearly every user that use my plugin don't use frontmatter. You are the first case in 2-3 years.
Adding a direct dependency would result in not used content for thousands of users.

Reply:
It brings sub-dependencies with it... https://github.com/jonschlinkert/gray-matter/blob/a5726b04f3167fadc764241deb545518c454eb82/package.json#L36-L41
And these have sub-sub-deps and so on.